articles/quickstart/native/android/06-calling-apis.md

```diff
@@ -22,23 +25,53 @@ This tutorial shows you how to use a previously saved token to authenticate your
 
 Before you continue with this tutorial, make sure that you have completed the previous tutorials. This tutorial assumes that:
 * You have completed the [Session Handling](/quickstart/native/android/03-session-handling) tutorial and you know how to handle the `Credentials` object.
-* You have set up a backend application as API. To learn how to do it, follow one of the [backend tutorials](https://auth0.com/docs/quickstart/backend). 
+* You have set up a backend application as API. To learn how to do it, follow one of the [backend tutorials](https://auth0.com/docs/quickstart/backend).
+
 
-After you set up an API, declare the endpoint you need to call as a constant in the current class:
+<%= include('_includes/_calling_api_create_api') %>
+
+<%= include('_includes/_calling_api_create_scope') %>__
+
+
+## Get the User's Access Token
+
+Get the user's access token ready for your API to consume by requesting as audience the API identifier used when creating the Auth0 API. Do so by using the `withAudience` method of the WebAuthProvider builder. For instructions, see the [Login](/quickstart/native/android/00-login) tutorial. Add at the top of the class the constants for the backend-app API secure URL and the API identifier.
```

Comment:
`Get the user's access token ready for your API to consume by requesting as audience the API identifier used when creating the Auth0 API. Do so by using the `withAudience` method of the WebAuthProvider builder.` ->
`To retrieve an access token that is authorized to access your API, you need to specify the API Identifier you created in the Auth0 dashboard.`

`Add at the top of the class the constants for the backend-app API secure URL and the API identifier.` ->
`At the top of the class add the constants for accessing the API, **API_URL** and **API_IDENTIFIER**

Reply:
I changed this a bit